modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -941,10 +945,18 @@ private ObjectDefinition _toObjectDefinition(
 					}
 				).put(
 					"update",
-					addAction(
-						ActionKeys.UPDATE, "putObjectDefinition",
-						permissionName,
-						objectDefinition.getObjectDefinitionId())
+					() -> {
+						if (!FeatureFlagManagerUtil.isEnabled("LPS-148856") &&
+							objectDefinition.isUnmodifiableSystemObject()) {
+
+							return null;
+						}
```

Comment:
I did not understand this code. Could you please remind me about this?

Reply:
Resolved